apps/docs/src/app/platform/component-docs/platform-file-uploader/platform-file-uploader-examples/platform-file-uploader-compact-example.component.html

```diff
@@ -0,0 +1,33 @@
+<fdp-form-group #fg>
+    <fdp-form-field
+        #ffl1
+        label="Default Input Field"
+        [id]="'input1'"
+        name="reactiveFormInput"
+        zone="“zLeft”"
+        rank="1"
+        [placeholder]="'Field placeholder text'"
+    >
+        <fdp-file-uploader
+            ariaLabel="Choose file"
+            ariaLabelledBy="Choose file"
+            placeholder="Choose File to upload"
+            buttonLabel="Browse..."
+            buttonAriaLabel="browse file"
+            accept=".png,.jpg"
+            [(ngModel)]="files"
+            multiple="true"
+            contentDensity="compact"
+            (selectionChange)="handleFileSelection($event)"
+        ></fdp-file-uploader>
+    </fdp-form-field>
+</fdp-form-group>
+<br /><br />
+
+Files Selected:
+<br />
+<span *ngFor="let file of this.files">
+    <span class="green">{{ file.name }} </span><br
```

Comment:
what is `class="green"`? Is there css for this?

Reply:
yes, its in the platform-file-uploader-example.component.scss, i have pointed all files to the same scss files. to avoid code redundancy

Reply:
this will give issues in stackblitz, each example should have css in its own file